packages/create-subscription/README.md

```diff
@@ -11,6 +11,16 @@ Other cases have **better long-term solutions**:
 * I/O subscriptions (e.g. notifications) that update infrequently should use [`simple-cache-provider`](https://github.com/facebook/react/blob/master/packages/simple-cache-provider/README.md) instead.
 * Complex libraries like Relay/Apollo should manage subscriptions manually with the same techniques which this library uses under the hood (as referenced [here](https://gist.github.com/bvaughn/d569177d70b50b58bff69c3c4a5353f3)) in a way that is most optimized for their library usage.
 
+## Compatibility with async mode
+
+The main motivation for `create-subscription` is to provide a way for library authors to ensure compatibility with React's upcoming asynchronous rendering mode. `create-subscription` guarantees correctness in async mode, accounting for the subtle bugs and edge cases that a library author might otherwise miss.
+
+However, it acheives correctness by sometimes de-opting to synchronous mode, obviating async's benefits. This is an inherent limitation of subscriptions—storing state outside of React's managed state queue and rendering in response to a change event—within React's async rendering model.
```

Comment:
> This is an inherent limitation of subscriptions—storing state outside of React's managed state queue and rendering in response to a change event—within React's async rendering model.

this sentence doesn't parse right to me

Reply:
Maybe:

>This is an inherent limitation of subscriptions in React: storing state outside of React's managed state queue and rendering in response to a change event is problematic with React's async rendering model.

Reply:
Confession: I hate the word problematic. So much baggage. Other than that, good suggestion.

Reply:
I initially went with "incompatible with React's async rendering model", but I wasn't sure if it was too strong of a statement. If it isn't, I like it better.